tests/cypress/integration/2-general-onboarding-flow/basic-info.cy.js

```diff
@@ -205,4 +211,24 @@ describe( 'Basic Info Page', function () {
 				} );
 		}
 	} );
+
+	it( 'Basic Info Events API', () => {
+		const label_keys = [
+			'',
+			'title',
+			'tagline',
+			'logo_added',
+			'platform',
+			'platform',
+		];
+		const actual_values = [ '', title, desc, '', 'facebook', 'instagram' ];
```

Comment:
Are we checking only for 2 social links?

Reply:
Added for all